clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -725,6 +732,26 @@ private static void monitorDataRecovery(CommandLine cmd) {
     }
   }
 
+  private static Set<String> calculateRecoveryStoreNames(String multiStores, String cluster, String parentUrl) {
+    if (!StringUtils.isEmpty(multiStores)) {
+      return Utils.parseCommaSeparatedStringToSet(multiStores);
```

Comment:
Do you think it would be helpful to print a message here when both cluster and stores are present, mentioning that '--stores' will overwrite '--cluster' values, or mention that in the data recovery command help text? Because users might think stores in both stores list and cluster will be executed.

Reply:
Yes, I think it is a good point. I will do both, i.e. printing out a warning message and displaying it in the command help text. Btw, do you think the which behavior is more intuitive in this case,  `overwrite` or `append`?

Reply:
Personally I think the right behavior is to respect stores parameter as you do in this PR. Because store is a finer grained resource than cluster. Other admin commands also execute on the store, instead of the whole cluster, when both are provided. Printing a message is to make the behavior explicitly clear to users.